plugins/diff/erofs/differ_linux.go

```diff
@@ -245,6 +268,20 @@ func (s erofsDiff) Apply(ctx context.Context, desc ocispec.Descriptor, mounts []
 	}
 	defer ra.Close()
 
+	layerBlobPath := path.Join(layer, "layer.erofs")
+	if native {
+		// TODO: prefer to use hardlink as an optimization here
```

Comment:
`io.Copy` in this case will usually do a `copy_file_range` which is better in this when reflinking is supported. We can also optimize this for the mac case to clone. I think we can remove this comment

Reply:
I will remove the comment.